build.gradle.kts

```diff
@@ -0,0 +1,107 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.gradle.internal.hash.Hashing
+
+plugins {
+    `kotlin-dsl`
+    `groovy-gradle-plugin`
+    id("org.jetbrains.kotlin.jvm") version "1.8.20"
+    `idea`
+
+    `signing`
+    `java-library`
+    `maven-publish`
+    id("com.gradle.plugin-publish") version "1.2.1"
+    id("com.palantir.git-version") version "3.0.0" apply false
+    id("com.github.johnrengelman.shadow") version "8.1.1" apply false
+    id("io.github.gradle-nexus.publish-plugin") version "1.3.0" apply false
+}
+
+group = "org.wiremock.tools.gradle"
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
+    }
+}
+
+repositories {
+    gradlePluginPortal()
+    mavenCentral()
+}
+
+gradlePlugin {
+    website = "https://github.com/wiremock/gradle-wiremock-extension-plugins"
+    vcsUrl = "https://github.com/wiremock/gradle-wiremock-extension-plugins"
+    plugins {
+        /*register("wiremock-extension-convention") {
```

Comment:
Just confirmed on Gradle Slack such pre-compiled script packaging is officially supported feature, so will delete